operator-common/src/main/java/io/strimzi/operator/common/model/Labels.java

```diff
@@ -242,6 +242,18 @@ public ResourceType type() {
         return type != null ? ResourceType.fromName(type) : null;
     }
 
+    public Labels strimziLabels() {
+        Map<String, String> newLabels = new HashMap<>(labels.size() + 1);
```

Comment:
out of curiosity, why the +1?

Reply:
Yeah ... good eyes. That is just bad copy paste programming. I changed it to 3 which is the usual number of `strimzi.io` labels we have currently